src/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+// Variables
+//
+// Variables should follow the `$component-state-property-size` formula for
+// consistent naming. Ex: $nav-link-disabled-color and $modal-content-box-shadow-xs.
+
+// $bootstrap-icons-font-src: url("../../node_modules/bootstrap-icons/font/fonts/bootstrap-icons.woff2") format("woff2"), url("../../node_modules/bootstrap-icons/font/fonts/bootstrap-icons.woff") format("woff");
+
+$info: #ffe7a6;
```

Comment:
I see you have a variable here but can't find where you used it

Reply:
It's the feed menubar over the add new post